src/main/kotlin/com/exactpro/th2/lwdataprovider/http/SseRequestContext.kt

```diff
@@ -67,15 +67,17 @@ class HttpMessagesRequestHandler(
         if (!isAlive) return
         val counter = indexer.nextIndex()
         val future: CompletableFuture<SseEvent> = data.completed.thenApplyAsync({ requestedMessage: RequestedMessage ->
-            if (jsonFormatter != null && requestedMessage.protoMessage == null && requestedMessage.transportMessage == null) {
-                builder.codecTimeoutError(requestedMessage.storedMessage.id, counter)
-            } else {
-                builder.build(
-                    requestedMessage, jsonFormatter, includeRaw,
-                    counter,
-                )
-            }.also {
-                HttpWriteMetrics.incConverted()
+            dataMeasurement.start("convert_to_json").use {
+                if (jsonFormatter != null && requestedMessage.protoMessage == null && requestedMessage.transportMessage == null) {
+                    builder.codecTimeoutError(requestedMessage.storedMessage.id, counter)
+                } else {
+                    builder.build(
+                        requestedMessage, jsonFormatter, includeRaw,
+                        counter,
+                    )
```

Comment:
I think we should measure only this part

Reply:
I would like to measure the whole part of our convener executed in a separate thread

Reply:
Okay